clayui.com/content/docs/components/css-icons.mdx

```diff
@@ -39,9 +54,7 @@ We use SVG elements that link to an SVG sprite, like so:
 
 Note that the ID after the # symbol is the ID of the icon to use, so if you wanted to use plus icon, you would do change the href to `path/to/icons.svg#plus`.
 
-<ul class="lexicon-icon-list list-unstyled">
-	[foreach Icons]
-</ul>
+<IconSearch source={icons} />
```

Comment:
Can we add an additional `## header` here and then add that link to the table of contents? I think it will help the flow and also will make this table linkable.